install.ps1

```diff
@@ -0,0 +1,132 @@
+﻿#
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#Requires -RunAsAdministrator
+
+<#
+ .SYNOPSIS
+    Installs OpenTelemetry .NET Automatic Instrumentation 
+ .PARAMETER InstallDir
+    Default: <auto> - the default value is AppData
+    Install path of the OpenTelemetry .NET Automatic Instrumentation
+    Possible values: <auto>, ProgramFiles, AppData, (Custom path)
+#>
+[cmdletbinding()]
+param(
+    [string]$InstallDir="<auto>"
+)
+
+function Get-Install-Directory([string]$InstallDir) {
+    $dir = "OpenTelemetry .NET AutoInstrumentation"
+    
+    if($InstallDir -eq "<auto>" -or $installDir -eq "AppData") {
+        return (Join-Path $env:LOCALAPPDATA "Programs" | Join-Path -ChildPath $dir)
+    }
+    elseif($InstallDir -eq "ProgramFiles"){
+        return (Join-Path "C:\Program Files\" $dir)
+    } 
+    elseif(Test-Path $InstallDir -IsValid) {
+        return $InstallDir
+    }
+
+    throw "Invalid install directory provided '$InstallDir'"
+}
+
+function Get-Temp-Directory() {
+    return $env:temp
+}
+
+function Prepare-Install-Directory([string]$InstallDir) {
+    New-Item -ItemType Directory -Force -Path $InstallDir | Out-Null
+}
+
+function Download-OpenTelemetry([string]$Version, [string]$Path) {
+    $archive = "opentelemetry-dotnet-instrumentation-windows.zip"
+    $dlUrl = "https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/download/$Version/$archive"
+    $dlPath = Join-Path $Path $archive
+
+    Invoke-WebRequest -Uri $dlUrl -OutFile $dlPath
```

Comment:
Have you tested the error handling?

I think the script will continue in case of an error

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1297/commits/f03ac2c70ac1e2227f134c54df1b34b443138087